data/json/vehicleparts/vehicle_parts.json

```diff
@@ -202,7 +213,7 @@
     "damage_modifier": 60,
     "durability": 95,
     "description": "A small but comfortable bed.",
-    "size": "50 L",
+    "size": "100  L",
```

Comment:
Surprised this passed tests you appear to have added an extra space after the number (More interested in tests catching this in future than a change to this specifically)

Reply:
Huh, yeah. I have fixed it in #70560 but that's something that shouldn't be getting through.